### ISSUE TITLE
Audio Collections: applied to training set

### ISSUE THREAD

**Issue by daebwae** [rfcx/rfcx-console#95]
The overall goal is to create training data for training of a deep learning model (DLM). 

Background: Think about DLM like teaching a student what a chainsaw sounds like. First, you would show them a couple of sound files with chainsaws (training set). The DLM learns what it thinks a chainsaw sound like. Then we also need to see whether DLM got it right. In a classroom, we would have an exam to test the student's understanding and grade the exam. The same is done for DLMs as well. We show the DLMs unheard chainsaw sounds  and other sounds. Then the DLM tell us what it thinks it heard. From the number of correct and wrong files, we can calculate a grade / score for the test. This way we know whether we have to show the DLM more files to further train it or whether it is ready to go out from the classroom and venture into the world to detect sounds in the rainforest. 

Audio Collection: for achieving this goal, we first define a new term an "audio collection". An audio collection has an id and is associated with **n** audio guids. It also contains a possible note for each audio added to the collection. The audio collections are ordered. This is a tabular representation of an audio collection: 

Collection id (integer and guid), collection position, audio guid, notes
1,0,51d12121-7279-433e-bc6f-871f0d068dbc, weak signal but significantly different
1,1,28636259-8df3-4693-90da-827efca403ca, 
2,0,dcd40af0-2548-4ad7-b286-4250b42593fb,
2,1,10cfcbb6-e27f-4a9d-8d26-250d47f8333a, strong signal - very representative
2,2,fa9b504f-7e64-4513-83d2-9cba50b6ccf6, wind  

I didn't add the guid for this example representation but in the DB, we need it. We need to create the audio collection in the database and models in sequelize to support this new concept. 

Workflow: A neural net specialist needs to select examples for training the neural net. To do that, they will (1) first create a new training data set or create a new one and then (2) add new training/test sets. 

Step 1 

![image](https://cloud.githubusercontent.com/assets/749854/19674057/92c147c0-9a85-11e6-8f80-6e6d7f0e74cc.png)

Here we can either select an existing training or create a new one. 

A training set has the following attributes: 
- Name
- Tag Value : chainsaw, amazon parrot, wind, ...
- Training set: this is an audio collection
- Test set: this is a second audio collection

Step 2 

![image](https://cloud.githubusercontent.com/assets/749854/19675373/44026804-9a8e-11e6-9cb6-01a662609533.png)

![image](https://cloud.githubusercontent.com/assets/749854/19675383/4ea40b82-9a8e-11e6-8ac8-3630b0cff3d4.png)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/167270113296771/201707224206373)
